canvas_modules/common-canvas/src/common-properties/ui-conditions/condition-ops/isNotEmpty.js

```diff
@@ -20,14 +20,15 @@ function op() {
 	return "isNotEmpty";
 }
 function evaluate(paramInfo, param2Info, value, controller) {
+	const propertiesConfig = controller.getPropertiesConfig();
 	const dataType = typeof paramInfo.value;
 	switch (dataType) {
 	case "undefined":
 		return false;
 	case "boolean":
 		return true;
 	case "string":
-		return paramInfo.value.trim().length !== 0;
+		return propertiesConfig.trimSpaces ? paramInfo.value.trim().length === 0 : paramInfo.value.length === 0;
```

Comment:
This is wrong. Thhe `isNotEmpty` test is checking for not empty
<img width="540" alt="image" src="https://user-images.githubusercontent.com/1628100/213829469-63dae559-1177-41dd-ad60-d2aa95335752.png">